api/proxy/store/generated_key/memstore/v2/memstore.go

```diff
@@ -180,39 +181,63 @@ func (e *MemStore) generateRandomCert(blobContents []byte) (coretypes.EigenDACer
 func (e *MemStore) Get(
 	_ context.Context, versionedCert certs.VersionedCert, returnEncodedPayload bool,
 ) ([]byte, error) {
-	encodedBlob, err := e.FetchEntry(crypto.Keccak256Hash(versionedCert.SerializedCert).Bytes())
+	blobSerialized, err := e.FetchEntry(crypto.Keccak256Hash(versionedCert.SerializedCert).Bytes())
+	if err != nil {
+		return nil, fmt.Errorf("fetching entry via memstore: %w", err)
```

Comment:
@samlaf not for this PR but we should probably define a detailed error for when a blob no longer exists